include/rocksdb/perf_context.h

```diff
@@ -225,9 +225,9 @@ struct PerfContext {
   uint64_t iter_seek_cpu_nanos;
 
   // Time spent in encrypting data. Populated when EncryptedEnv is used.
-  uint64_t encrypt_data_nanos;
+  uint64_t encrypt_data_nanos = 0;
   // Time spent in decrypting data. Populated when EncryptedEnv is used.
-  uint64_t decrypt_data_nanos;
+  uint64_t decrypt_data_nanos = 0;
```

Comment:
Why are these different than the other values?  Should they just be added to the Reset() method instead?